packages/remix-dev/vite/plugin.ts

```diff
@@ -255,12 +255,20 @@ export const remixVitePlugin: RemixVitePlugin = (options = {}) => {
 
   let resolvePluginConfig =
     async (): Promise<ResolvedRemixVitePluginConfig> => {
+      let defaults: Partial<RemixVitePluginOptions> = {
+        serverBuildPath: "build/server/index.js",
```

Comment:
Reminder to self: Check whether the old defaults are being shown via JSDoc comments.

Reply:
Fixed by https://github.com/remix-run/remix/pull/8077/commits/039f8dfa8e1b21edc0c3aef2bfc4ddfed32a7d54.